data/mapcss-mapping.csv

```diff
@@ -582,7 +582,7 @@ railway|monorail|bridge;[railway=monorail][bridge?];;name;int_name;469;
 area:highway|steps;470;
 # ~90k usages.
 amenity|telephone;471;
-deprecated|deprecated;472;x
+tourism|camp_pitch;472;
```

Comment:
@vng is it the right ID for the pitch?

Reply:
Thanks -- I was just trying to follow instructions suggesting to reuse deprecated ids when possible.